test/addon_runners/chrome/data/panel.js

```diff
@@ -1,3 +1,4 @@
+/* jshint ignore: start */
```

Comment:
I did that because this file was very short, and I didn't understand where `self` was defined, but I didn't want to change anything.

Note that in this branch, I'm excluding `test/addon_runners` from jshint checks because I couldn't get jshint's "esversion: 6" option to recognize the new `=>` syntax used throughout this code.